special-interest-groups/sig-docs/roles-and-organization-management.md

```diff
@@ -0,0 +1,68 @@
+# Docs SIG Roles and Organization Management
+
+<!--
+Outline the organism specific to this SIG, as well as those that differ from [SIG Governance](sig-governance.md)
+-->
+
+## Join and Promotion
+
+We mainly follow the instructions in [TiDB Developer Group](../../architecture/README.md#tidb-developer-group), including common requirements, responsibilities and privileges of each role in the developer group.
+
+### Join
+
+You are welcomed to be a [contributor](../../architecture/README.md#contributor) once one of your [PR about documentation](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md) is merged. To have a discussion with us, just join our Slack channel **#sig-docs** in the [TiDB Community](https://pingcap.com/tidbslack) Slack workspace.
+
+### Promotion
+
+#### Be Promoted to [`Active Contributor`](../../architecture/README.md#active-contributor)
+
+SIG Specific Requirements:
+
+* Joined the [#sig-docs](https://tidbcommunity.slack.com/messages/sig-docs) Slack channel
```

Comment:
```suggestion
* Joined the [#sig-docs](https://tidbcommunity.slack.com/messages/sig-docs) Slack channel
```
No other requirements for Active Contributor?

Reply:
This is an additional item on the basis of the general requirements. But it might be clearer to put those general requirements here. I'll add them here.

Reply:
Added.